packages/service-integration/src/service_integration/osparc_config.py

```diff
@@ -82,7 +85,7 @@ def create_default(cls, service_name: str = None) -> "DockerComposeOverwriteCfg"
     def from_yaml(cls, path: Path) -> "DockerComposeOverwriteCfg":
         with path.open() as fh:
             data = yaml_safe_load(fh)
-        return cls.parse_obj(data)
+            return cls.parse_obj(data)
```

Comment:
Question: Just curious, any particular reason for doing this?

Reply:
there was no need. reverting